package-lock.json

```diff
@@ -45,6 +45,21 @@
             "ms": "2.1.2"
           }
         },
+        "json5": {
```

Comment:
Do we need to submit this file?

Reply:
I did an `rm -f node_modules` and a `npm install` to see if this changed the package-lock file, but nothing was added or removed.